custom_components/luxtronik/sensor.py

```diff
@@ -351,6 +352,16 @@ async def async_setup_entry(
                 "mdi:waves-arrow-right",
                 entity_category=None,
             ),
+            LuxtronikSensor(
+                hass,
+                luxtronik,
+                device_info_heating,
+                "calculations.ID_WEB_Temperatur_TRL_ext",
+                "flow_out_temperature",
```

Comment:
This key is already used for "calculations.ID_WEB_Temperatur_TRL".
Please change it.